### ISSUE TITLE
Docs TS: HTTP Client

### ISSUE THREAD

**Issue by Foxandxss** [angular/angular-es#19]

**Comment by marinhero** [comment]
Hello! does this makes reference to this page? https://docs.angularjs.org/api/ng/service/$http

**Comment by Foxandxss** [comment]
No, that is Angular 1 docs.

**Comment by marinhero** [comment]
Ooops! must be this one then: https://angular.io/docs/ts/latest/tutorial/toh-pt6.html right? if so, can I have it ?

**Comment by Foxandxss** [comment]
No, is this one: https://angular.io/docs/ts/latest/guide/server-communication.html

Yes, you can have it.

Follow the conventions done in the chinese repo or ask if you have any question.

**Comment by marinhero** [comment]
All right Jesus, thanks! I'll reach out when the questions appear :)
